acquia_lift.admin.wizard.inc

```diff
@@ -299,6 +296,20 @@ function acquia_lift_personalize_campaign_wizard_targeting_alter(&$form, &$form_
     $variation_options[$option['option_id']] = $option['option_label'];
   }
 
+  // The "Everyone else" audience should be anything explicitly assigned plus
+  // any options that aren't assigned.
```

Comment:
Per our discussion, the "everyone else" audience needs to be created up front and all variations assigned to it - this would happen when the variation set is created.